dist/profile/manifests/jenkinscontroller.pp

```diff
@@ -334,31 +335,10 @@
     require => File[$docroot],
   }
 
-  apache::vhost { $ci_fqdn:
-    serveraliases         => [
-      # Give all our jenkinscontroller profiles this server alias; it's easier than
-      # parameterizing it for compatibility's sake
-      'ci.jenkins-ci.org', $ci_resource_domain,
-    ],
-    require               => [
-      Docker::Run[$docker_container_name],
-      File[$docroot],
-      # We need our installation to be secure before we allow access
-      File[$groovy_d],
-    ],
-    port                  => 443,
-    override              => 'All',
-    ssl                   => true,
-    docroot               => $docroot,
-
-    access_log_pipe       => "|/usr/bin/rotatelogs -t ${apache_log_dir}/access.log.%Y%m%d%H%M%S 86400",
-    error_log_pipe        => "|/usr/bin/rotatelogs -t ${apache_log_dir}/error.log.%Y%m%d%H%M%S 86400",
-    proxy_preserve_host   => true,
-    allow_encoded_slashes => 'on',
-    custom_fragment       => "
+  $base_custom_fragment = "
 RequestHeader set X-Forwarded-Proto \"https\"
 RequestHeader set X-Forwarded-Port \"${proxy_port}\"
-RequestHeader set X-Forwarded-Host \"${ci_fqdn}\"
+RequestHeader set X-Forwarded-Host \"${ci_resource_domain}\"
```

Comment:
Is that the wrong domain?

Reply:
Yep, that line might have to be separated in another custom fragment

Reply:
I've separated it, I missed that one, good catch!